src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -152,39 +175,93 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                     <Typography variant="h2" gutterBottom>
                       {t('registration.files_and_license.files')}
                     </Typography>
+
                     {files.length > 0 && (
                       <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mb: '2rem' }}>
-                        {files.map((file) => {
-                          const associatedFileIndex = associatedArtifacts.findIndex((artifact) => {
-                            if (associatedArtifactIsFile(artifact)) {
-                              const associatedFile = artifact as AssociatedFile;
-                              return associatedFile.identifier === file.identifier;
-                            }
-                            return false;
-                          });
+                        <TableContainer component={Paper}>
```

Comment:
Hm, vi ønsker at alle tabeller skal vises likt, så ideelt sett skulle vi satt `component={Paper}` i theme eller noe. Mener jeg har prøvd å få til noe sånt før, uten å lykkes. Tabellen for contributors ser feks annerledes ut nå, siden den ikke setter Paper eksplisitt. Mulig den enkleste løsningen her er å sette Paper eksplisitt over alt. Kan evt notere deg det som en oppgave til en egen PR etterpå

Reply:
Syns tabllen ble mye finere med en slik border iaf